packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -668,22 +668,3 @@ class FlutterTask extends BaseFlutterTask {
     }
 }
 
-gradle.useLogger(new FlutterEventLogger())
```

Comment:
why removing this?

Reply:
The Logger is currently eating important errors when the native build fails. You can try writing invalid XML in a file within res/main/values file to see how the error message will not be shown anymore.

This change should not be included in the PR. I will remove and and submit a separate PR.